lib/rialto/etl/cli/publications.rb

```diff
@@ -42,39 +54,44 @@ def load
           CSV.foreach(csv_path, headers: true, header_converters: :symbol) do |row|
             count += 1
             next if offset > count
-            load_row(row, count)
+            load_row(row, count, options[:skip_existing], options[:skip_load])
```

Comment:
Can you rename 'load_row'. it seems odd to me that we'd pass `:skip_load` to a method that purportedly does the loading.

Reply:
Fixed.